### PR TITLE
help: Add flags to output, ssh category

### PR DESCRIPTION
**Pull request by yedayak**
Add --output, --remove-on-error, --output-dir and --created-dirs to the output help category
Add --hostpubmd5, --hostpubsha256, --insecure (-k) and --pubkey to the ssh help category

### CODE REVIEW

**Comment by vszakats** [comment]
Perhaps the new categories should also be added to the corresponding `.md` manuals' `Category:` line in `docs/cmdline-opts/`?

**Comment by yedayak** [comment]
> Perhaps the new categories should also be added to the corresponding `.md` manuals' `Category:` line in `docs/cmdline-opts/`?

Sure, did that and force pushed

**Review by bagder — APPROVED** [review]

**Review by vszakats — APPROVED** [review]

**Comment by bagder** [comment]
Maybe `--output-dir` should go into **output** as well?

How about `--create-dirs` ?

**Comment by yedayak** [comment]
> Maybe `--output-dir` should go into **output** as well?
> 
> How about `--create-dirs` ?

Agreed, added those two

**Comment by yedayak** [comment]
I added two more commits that add options to the ssh category, and mention the exact command needed to generate the `tool_listhelp.c`

File: scripts/managen [hunk 1]
```diff
@@ -918,7 +918,7 @@ sub listhelp {
  * This source file is generated with the following command in an autotools
  * build:
  *
- * "make listhelp"
+ * "make -C docs/cmdline-opts/ listhelp"
```
Comment (vszakats) [inline scripts/managen:921]:
```suggestion
 * "make -C docs/cmdline-opts listhelp"
```
For consistency with other similar lines, e.g. in `src/tool_help.h`.

File: src/tool_listhelp.c [hunk 2]
```diff
@@ -29,7 +29,7 @@
  * This source file is generated with the following command in an autotools
  * build:
  *
- * "make listhelp"
+ * "make -C docs/cmdline-opts/ listhelp"
```
Comment (vszakats) [inline src/tool_listhelp.c:32]: Also here.

Comment (bagder) [reply src/tool_listhelp.c:32]: I can generate the source fine with `make listhelp` exactly as already documented.

Comment (yedayak) [reply src/tool_listhelp.c:32]: > I can generate the source fine with `make listhelp` exactly as already documented.
> 
> I invoke that command in the `src/` directory.

OK, that makes sense, I didn't know `src/` had it's own targets. Will remove this commit.

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**